### DISCUSSION TITLE
Get-Help on Linux (PowerShell Core 7.1) missing documentation

### DISCUSSION THREAD

**Discussion by rvdginste** [dataplat/dbatools#7947] [Q&A] [answered]
Hello,

I've been using dbatools for a while and think it's an amazing tool. Big thanks to all contributors!

Something that I've noticed for a while is that on linux the documentation seems to be missing. I can use the Get-Help command but it does not have the 'written' documentation, for example it misses the 'synopsis', the examples, and so on. Is there something I need to do to get that documentation?

I installed dbatools using "install-module dbatools" in powershell core.

Thanks,
Ruben

**Comment by potatoqualitee** [comment, marked as answer]
Whoa, thank you for reporting this. I can repro on PS 7.3 on LInux as well. @StartAutomating you may want to update your module. From my testing, PS7 on Linux and PS on Windows, both core and desktop, look in different places :( So now in my pipeline release, I copy dbatools-Help.xml to the root. Then it works on both Windows and Linux.

**Reply by rvdginste** [reply]
No problem, I thought it was an issue on my side.
I see that it's working fine now on the latest version, thanks!
